src/foam/nanos/controller/ApplicationController.js

```diff
@@ -752,7 +752,7 @@ foam.CLASS({
           await this.fetchSubject();
           return;
         }
-        menu = await this.findFirstMenuIHavePermissionFor(dao);
+        await this.routeTo(menu?.id || '');
```

Comment:
```suggestion
        await this.pushDefaultMenu();
```

Reply:
what if the theme doesn't have a default menu?

Reply:
Then the theme should be fixed.